changelogs/unreleased/gh-7593-console-debugger.md

```diff
@@ -0,0 +1,6 @@
+## feature/debugger
+
+* Created a console debugger `luadebug.lua` for debugging of external and
+  builtin Lua modules;
+* New Lua global function created `tarantool_builtin_module` which allows
```

Comment:
Nit: don't forget to mention the right name in changelog.

Reply:
Done. Thanks!